org.lflang/src/org/lflang/generator/python/PythonPortGenerator.java

```diff
@@ -195,7 +195,7 @@ public static String generatePythonListForContainedBank(String reactorName, Port
             "for (int i = 0; i < "+generateWidthVariable(reactorName)+"; i++) {",
             "    if (PyList_SetItem("+reactorName+"_py_list,",
             "            i,",
-            "            "+generateConvertCPortToPy("self->_lf_"+reactorName+"[i]."+port.getName(), widthSpec),
+            "            "+generateConvertCPortToPy((port.getWidthSpec() == null ? "&" : "") + "self->_lf_"+reactorName+"[i]."+port.getName(), widthSpec),
```

Comment:
@Soroosh129 I had to make this change in order to get my test models to compile. Otherwise, I got the following error from gcc:
```
error: incompatible type for argument 1 of ‘convert_C_port_to_py’
expected ‘void *’ but argument is of type ‘countercycle_next_t’ {aka ‘struct <anonymous>’}
  418 | PyObject* convert_C_port_to_py(void* port, int width) {
```

This occurred for every reaction that had an effect on a bank of reactors but only if the port was not a multiport.

Can you confirm that this fix is correct or is the actual problems somewhere else?

Reply:
I think a simpler potential solution would be to use `reactorName + "[i]." + port.getName()` instead of `"self->_lf_"+reactorName+"[i]."+port.getName()`.

Reply:
That looks like a better solution. Thanks!